packages/dso-toolkit/src/components/alert/alert.scss

```diff
@@ -1,47 +1,67 @@
 @use "../alert";
-
 @use "../../di";
+@use "../../variables/units";
```

Comment:
Deze Sass module wordt in hier niet gebruikt

Reply:
Gedaan